cli/loadCompletion.sh

```
@@ -0,0 +1,43 @@
+  }
+  compdef _opfab_completion opfab
+elif type complete &>/dev/null; then
+  _opfab_completion() {
+    local cur prev nb_colon
+    _get_comp_words_by_ref -n : cur prev
+    nb_colon=$(grep -o ":" <<< "$COMP_LINE" | wc -l)
+
+    COMPREPLY=( $(compgen -W '$(opfab --compbash --compgen "$((COMP_CWORD - (nb_colon * 2)))" "$prev" "${COMP_LINE}")' -- "$cur") )
+    __ltrim_colon_completions "$cur"
+    # Added 
+    [[ $COMPREPLY ]] && return      
```

Shellcheck

[warning] Expanding an array without an index only gives the first element.

(SC2128)